code/extensions/RealMeSecurityExtension.php

```diff
@@ -91,17 +94,50 @@ public function realme()
      */
     private function realMeACS()
     {
-        $loggedIn = $this->service->enforceLogin();
+        try {
+            $authenticated = $this->service->enforceLogin();
+
+            if($authenticated === true) {
+                $authData = $this->service->getAuthData();
+
+                // If more session vars are set here, they must be cleared in realmeLogout()
+                Session::set('RealMeSessionData', serialize($authData));
+                Session::set('RealMeOriginalResponse', $_POST['SAMLResponse']);
```

Comment:
@dhensby Do you have any suggestions as to what context to sanatise for?

In the current build, it seems like this was assigned to the session for posterity. Might it be worthwhile simply naming the reference more appropriately, so that users can sanatise for the correct case when using the SAMLResponse? e.g `Session::set('RealMe.RawSAMLResponse', $_POST['SAMLResponse']);`